tests/python/contrib/test_hexagon/test_vtcm_bandwidth.py

```diff
@@ -95,16 +95,17 @@ def evaluate(hexagon_session, sch, size):
     )
 
     # These are reduced for CI but number=100 and repeat=10 does a good job of removing noise.
-    number = 1
-    repeat = 1
+    number = 100
```

Comment:
I see this is a draft but make sure we do not push these changes as they would run unnecessarily slow in CI

ie
```
    if tvm.testing.utils.IS_IN_CI:
        # Run with reduced number and repeat for CI
        timer = module.time_evaluator("__tvm_main__", hexagon_session.device, number=1, repeat=1)
    else:
        timer = module.time_evaluator("__tvm_main__", hexagon_session.device, number=10, repeat=10)
```

Reply:
Yes indeed.  I will clean all this up before publishing the PR.